filter.go

```diff
@@ -42,9 +36,15 @@ func FilterCheck(results []*CheckResult, diff []*diff.FileDiff, strip int, wd st
 
 	return checks
 }
-
-func cleanPath(path string) string {
-	p := filepath.Clean(path)
+func CleanPath(path, workdir string) string {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function CleanPath should have comment or be unexported